### DISCUSSION TITLE
Unable to access Nuxt Context?

### DISCUSSION THREAD

**Discussion by sei-jdshimkoski** [danielroe/nuxt-vitest#166] [Q&A]
Hello all,

I am trying to do a simple test just to get this working but I am having no luck getting the Nuxt Context.

* Nuxt Context doesn't seem to work, for example, plugins don't seem to be recognized
* Using a value from the Nuxt Context in a template just throws an error
* I get notices about missing components that are added via a plugin
* I get a weird `/blank` route doesn't exist error
* Using the provided MockComponent appears to do nothing

Its a real mess. 

Is this module stable for anyone that is doing any sort of slightly more complex testing?

Below is a really dumb test just to see if I can get the Nuxt Context working but it just throws errors.

```javascript
// @vitest-environment nuxt
import { describe, test, expect } from 'vitest'
import { mountSuspended } from 'vitest-environment-nuxt/utils'
import Component from '@/components/Component.vue'

describe('Component.vue', () => {
  test('dumb test', async () => {
    const component = await mountSuspended(Component)
    expect(component.html()).toBeTruthy()
  })
})
```

**Comment by lassediercks** [comment]
We're working on a migration of a production site to nuxt3 and are using nuxt-vitest, is your scenario working in an isolated environment? We experienced some npm packages that haven't updated to vue2 to be a real problem for the test suite
